driver-core/src/test/functional/com/mongodb/operation/FindAndUpdateOperationSpecification.groovy

```diff
@@ -196,7 +294,22 @@ class FindAndUpdateOperationSpecification extends OperationFunctionalSpecificati
         async << [true, false]
     }
 
-    @IgnoreIf({ !serverVersionAtLeast(3, 2) })
+    @IgnoreIf({ !serverVersionAtLeast(asList(4, 1, 11)) })
```

Comment:
Perhaps test other failure modes - a List of BsonInt32 would also cause an error. Lets test that and the error.  It maybe fine just to return the server error or you may want to add a try catch to the toBsonDocumentList method and add a friendly error message.